jupyter_ydoc/ydoc.py

```diff
@@ -131,19 +271,51 @@ def get_cell(self, index: int) -> Dict[str, Any]:
             del cell["attachments"]
         return cell
 
-    def append_cell(self, value: Dict[str, Any], txn=None) -> None:
+    def append_cell(self, value: Dict[str, Any], txn: Optional[Y.YTransaction] = None) -> None:
+        """
+        Adds a cell.
+
+        :param value: A cell.
+        :type value: Dict[str, Any]
+
+        :param txn: A YTransaction, defaults to None
+        :type txn: :class:`YTransaction` | None, optional.
```

Comment:
I think it's redundant to have `| None` and `optional`.
```suggestion
        :type txn: :class:`YTransaction`, optional.
```